### DISCUSSION TITLE
Boot from an external USB or Hard Drive in macOS

### DISCUSSION THREAD

**Discussion by SpecterGames** [utmapp/UTM#6590] [Linux]
I saw another feature request of this same topic but it was on iOS, but this feature will be implemented in macOS as there is no feature to this yet. I connected the hard drive to the VM by USB passthrough, then going to the boot manager and booting it from there but no luck as it just comes back to the boot manager. Any recommendations or is there just no features?

**Comment by SpecterGames** [comment]
UPDATE 4/7/2024 8:32PM

I tried flashing an ISO of Arch Linux (x86_64) and booting it from the boot manager and surprisingly it worked for some reason. If that worked, then why didn't the OS boot from my external drive? (OS was Arch Linux x86_64, tried it on the x86_64 VM)

**Comment by osy** [comment]
As you found out with flashing an ISO, I think it should work but you have to configure your Linux, grub, etc correctly which is out of scope.
